src/compiler/symb.bi

```diff
@@ -1865,14 +1871,40 @@ declare function symbGetDefaultParamMode _
 
 declare function symbVarCheckAccess( byval sym as FBSYMBOL ptr ) as integer
 
+declare function symbCheckConstAssignTopLevel _
```

Comment:
Do we even need the special case versions to be public anymore? Can't all code just use the new generic `symbCheckConstAssign()`?

Reply:
haha, I don't know.  After 4 weeks of looking at the same 12 procs everyday, my brain was in knots.  I did try symbCheckConstAssign() as the generic initial proc at some point in my efforts, but at the time I tried it, I had infinite recursion and couldn't sort out how to solve it.  That's why there was some leftover checks for function types in a couple procs.  I think I can make another attempt at solving this out... and not go insane.